zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProvider.java

```diff
@@ -70,6 +70,7 @@
     private Server server;
     private final MetricsServletImpl servlet = new MetricsServletImpl();
     private final Context rootContext = new Context();
+    public final CustomizedExports customizedExports = CustomizedExports.instance();
```

Comment:
why "public" ?
it is initialised in `configure` and you can stop it in the close method

no need to do something special in the tests

Reply:
I need to clean up customizedExports in the PrometheusMetricsProviderTest#setup manually because customizedExports is the text information which will pollute all the existing UTs in the PrometheusMetricsProviderTest